openproblems/tasks/chromatin_potential/methods/linear_regression.py

```diff
@@ -0,0 +1,155 @@
+import numpy as np
+import pandas as pd
+from scipy.sparse import csr_matrix
+import scanpy as sc
+import os
+from ....tools.decorators import method
+
+
+def _chrom_limit(x):
+    y = x.values
+    if y[-1] == "+":
+        return [y[-3] - 1e5, y[-3] + 1e5]
+    else:
+        return [y[-2] - 1e5, y[-2] + 1e5]
+
+
+def _get_annotation(adata):
+    from pyensembl import EnsemblRelease
+
+    os.system("pyensembl install --release 100 --species mus_musculus")
```

Comment:
Can we at the very least use subprocess here instead of os.system? Better still, does `pyensembl` has a python API for this command?

Reply:
I can use subprocess, alternatively, I'll check the API from pyensembl to see if it is supported.